lib/template-loader.php

```diff
@@ -422,7 +422,7 @@ function gutenberg_template_loader_filter_block_editor_settings( $settings ) {
 	}
 
 	// If this is the Site Editor, auto-drafts for template parts have already been generated
-	// through `gutenberg_find_template_post_and_parts` in `gutenberg_edit_site_init`.
+	// through `gutenberg_find_template_post_and_parts`, when called via the REST API.
```

Comment:
This is not very clear. I know because I worked on it, but what's being called via the REST API, where, and when?

Reply:
Maybe "through `filter_rest_wp_template_part_query`," would be more clear?

Reply:
0c320da5ee

Reply:
I saw you added it as: 
```// through `gutenberg_find_template_post_and_parts`, when called through `filter_rest_wp_template_part_query`.```

Technically, `gutenberg_find_template_post_and_parts` isn't used there.  To clarify my earlier suggestion:
```// through `filter_rest_wp_template_part_query`, when called via the REST API.```

Reply:
ofc :man_facepalming: I'll change it